src/Apps/Order/OrderApp.tsx

```diff
@@ -8,13 +16,39 @@ export interface OrderAppProps {
     orderID: string
   }
   location: any
+  routeIndices: number[]
+  routes: any
+  router: any
```

Comment:
I think that Found actually comes with TS type defs, no?

Reply:
https://github.com/4Catalyzer/found/blob/master/index.d.ts

Reply:
🙀 

Okay. I'll take a look.

Reply:
Looks like it could do with some improvement to the `Matcher` API, though. Let me know if you want to pair on a PR.

Reply:
This PR or the found Matcher types?

Reply:
The Matcher types